CMakeLists.txt

```diff
@@ -754,10 +754,6 @@ endif()
 if(NOT Boost_FOUND)
     set(ONLY_SIMPLE ON)
     message(STATUS "Only building executable with few command-line options because the boost program_options library were not available")
-else()
-    message(STATUS "Boost -- found at library: ${Boost_LIBRARIES}")
-    message(STATUS "Boost -- adding '${Boost_LIBRARY_DIRS}' to link directories")
```

Comment:
I don't understand why this and the previous is removed. This is needed so we know what is being linked. There can be more than one Boost installation.

Reply:
CMake already prints information about found library. So I thought it's okay to remove it since `link_directories` is no longer necessary. I can put it back if you wish. But usually libraries doesn't print such information manully.